### DISCUSSION TITLE
Tasmota + Sonoff Zbbridge Pro Erratic Ping/HTTP Response Times

### DISCUSSION THREAD

**Discussion by mouldybread** [arendst/Tasmota#21279] [Support request] [answered]
Running 13.4.1.2 (582ca59-zbbrdgpro) on a Sonoff Zbbridge Pro.

While investigating Zigbee2MQTT reporting "Adapter disconnected" I discovered that my SonoffZigbeeBridge running Tasmota has erratic http response and ping times that are an order of magnitude higher than vanilla Tasmota running on a Sonoff switch.

UptimeKuma reports spikes in http response time every 5-10 minutes(>4000ms)when checking at a 60 second interval. Roughly every 4th and 5th ping have a >132ms response time, while the rest are between 50-100ms which is significantly higher than my other Tasmota devices.

Wifi RSSI 100%, -37 dBm. Other Tasmota devices have a normal/lower ping + http response time. So I don't think this is a network issue.

Anecdotally I did notice the web interface was sluggish but the only reason I'm digging is that I believe it may be related to the  Zigbee2MQTT disconnects. This device was previously working and stable, I'd say this has been an intermittent issue since January.

Any ideas on what might be happening here, or is this just normal for Tasmota running on this device? TIA.

**Comment by sfromis** [comment, marked as answer]
On ESP32, I'm also seeing "surprisingly long" ping times until I set `Sleep 0`. I suppose that this may use a bit more power.

**Reply by s-hadinger** [reply]
You can also try `Sleep 5`, it does improve things for me

**Reply by sfromis** [reply]
It does not for me. Only zero helped.

**Reply by mouldybread** [reply]
Fantastic, `Sleep 0` worked perfectly. Thank you so much!

**Reply by s-hadinger** [reply]
You can also use `SetOption127 1` to force wifi to not sleep, while keeping `Sleep 10` for example. This reduces power consumption and temperature of ESP32

**Reply by sfromis** [reply]
Doing a small test with `Ping8` from another Tasmota device, I saw no clear difference with `SetOption127 1`, when keeping `Sleep 50`.
